addons/web/static/src/core/domain_selector/domain_selector.xml

```diff
@@ -2,85 +2,94 @@
 <templates xml:space="preserve">
 
     <t t-name="web._DomainSelector">
```

Comment:
rename this template, it was like this because of the duplication with legacy

Reply:
done